game/state.py

```diff
@@ -4,22 +4,82 @@ def __init__(self, width, height):
         self.organisms = []
         self.width = width
         self.height = height
+        self.foods = []
 
 
     def transition(self, actions):
-        pass
+        for org in self.organisms:
+            for foo in self.foods:
+                eat_food = list(set(org.cells) & set(foo.cells))
```

Comment:
this is slick.  I would change it to this though:

    overlapping_cells = set(org.cells) & set(foo.cells)
    if overlapping_cells:
        org.foodstore += 1
        foo.delete_cells(eat_food[0])

check this out for why you don't have to say `if len(thing) > 0`
https://docs.python.org/2.4/lib/truth.html